backtesting/backtesting.py

```diff
@@ -1420,9 +1420,12 @@ def _optimize_skopt() -> Union[pd.Series,
 
             # np.inf/np.nan breaks sklearn, np.finfo(float).max breaks skopt.plots.plot_objective
             INVALID = 1e300
+            skopt_pbar = _tqdm(dimensions, total=max_tries, desc="Skopt optimizations")
```

Comment:
Let's call this description `'Backtest.optimize'` as this is the method the user sees running. And add a similar description for grid tqdm loop above.